components/dashboard/src/configurations/detail/ConfigurationDelete.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the repository root for license information.
+ */
+
+import { Project } from "@gitpod/gitpod-protocol";
+import { useCallback, useState } from "react";
+import { useHistory } from "react-router";
+import { Heading2, Subheading } from "../../components/typography/headings";
+import { RemoveConfigurationModal } from "./ConfigurationDeleteModal";
+
+interface RepositoryPrebuildsSettingsProps {
+    configuration: Project;
+}
+
+export default function DeleteConfiguration({ configuration }: RepositoryPrebuildsSettingsProps) {
+    const [showRemoveModal, setShowRemoveModal] = useState(false);
+
+    const history = useHistory();
+
+    const onProjectRemoved = useCallback(() => {
+        history.push("/repositories");
```

Comment:
This url doesn't work with it being changed to `/configuration` - but I think we should leave it as `/repositories` anyways.